example/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, writeText, nodejs
+# from purescript-nix
+, buildPackageLock, buildSpagoLock, purs, esbuild }:
+let
+  npmDependencies = buildPackageLock { src = ./.; };
+  workspaces = buildSpagoLock { src = ./.; };
+  entrypoint = writeText "entrypoint.js" ''
+    import { main } from "./output/App.Main";
+    main();
+  '';
+in stdenv.mkDerivation rec {
```

Comment:
This file is an example of how an end user would actually build their PureScript projects using the utilities provided by this library. While I want to stay low-level, it would be nice to provide suitable utilities such that a simple PureScript application can be packaged via Nix with just a few lines of code.